File: python/problems_01_25/problem_01.py
def sum_three_five(limit):
    """
    find all integers less than the limit that are
    multiples of 3 or 5 and return their sum
    """
    the_sum = 0
    for i in range(1,limit):
        if (i % 3 == 0) or (i % 5 == 0):
            the_sum += i
    return the_sum

File: node/problems_01_25/problem_01.js
// return the sume of all numbers less than the limit which are
// multiples of three or five
function SumThreeFive(limit) {
    var sum = 0;
    for (i=1; i < limit; i++) {
        if (i % 3 == 0 || i % 5 ==0 ){
            sum += i;
        }
    };
    return sum;
};

module.exports.SumThreeFive = SumThreeFive;
